### ISSUE TITLE
substrate node template monthly-2021-12 fails to build

### ISSUE THREAD

**Issue by justinFrevert** [substrate-developer-hub/substrate-node-template#289]
The `monthly-2021-12` tag of the substrate node template gives this error when attempting to build 

```
error: failed to select a version for `serde_json`.
    ... required by package `sc-cli v0.10.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2021-12#b6c1c1bc)`
    ... which satisfies git dependency `sc-cli` of package `frame-benchmarking-cli v4.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2021-12#b6c1c1bc)`
    ... which satisfies git dependency `frame-benchmarking-cli` of package `node-template v4.0.0-dev (/Users/justin/dev/new/substrate-node-template-monthly-2021-12/node)`
versions that meet the requirements `^1.0.71` are: 1.0.73, 1.0.72, 1.0.71

all possible versions conflict with previously selected packages.

  previously selected package `serde_json v1.0.68`
    ... which satisfies dependency `serde_json = "^1.0.59"` (locked to 1.0.68) of package `cargo_metadata v0.14.1`
    ... which satisfies dependency `cargo_metadata = "^0.14.1"` of package `substrate-wasm-builder v5.0.0-dev (https://github.com/paritytech/substrate.git?tag=monthly-2021-12#b6c1c1bc)`
    ... which satisfies git dependency `substrate-wasm-builder` of package `node-template-runtime v4.0.0-dev (/Users/justin/dev/new/substrate-node-template-monthly-2021-12/runtime)`
    ... which satisfies path dependency `node-template-runtime` (locked to 4.0.0-dev) of package `node-template v4.0.0-dev (/Users/justin/dev/new/substrate-node-template-monthly-2021-12/node)`

failed to select a version for `serde_json` which could resolve this conflict
```

**Comment by NukeManDan** [comment]
duplicate #282 , I will bump the `lastest` tag with a the fix and also `main` here.

**Comment by al3mart** [comment]
closed by #290
